src/types/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright(c) 2006 to 2019 ADLINK Technology Limited and others
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v. 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0, or the Eclipse Distribution License
+# v. 1.0 which is available at
+# http://www.eclipse.org/org/documents/edl-v10.php.
+#
+# SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+#
+add_subdirectory(tests)
+
+# Find Bison and Flex
+find_package(BISON REQUIRED)
+find_package(FLEX REQUIRED)
+find_package(Abstraction REQUIRED)
```

Comment:
Just a note: this is no longer needed with @k0ekk0ek's latest cmake modifications